### ISSUE TITLE
Add ability to disable use of Hotkeys / Desktop View

### ISSUE THREAD

**Issue by benz145** [mzomparelli/zVirtualDesktop#204]
For users only using one primary feature or other, it would be useful to be able to disable the other feature. Personally I'm not using Hotkeys right now, and would like to disable them so that I don't accidentally trigger anything without knowing it.

It looks like one could delete all of the keybindings to effectively disable the function but of course this isn't an ideal approach. With that said, a list of default hotkeys (separate from the active list), with the option to restore all of them or just one at a time would probably be useful.

**Comment by mzomparelli** [comment]
Simply turn off the mouse and keyboard hooks and no hotkeys will work. Also, I've been thinking of creating profiles for the hotkeys. With this you'd be able to create a profile with no key bindings.

![image](https://user-images.githubusercontent.com/10564234/42217694-c9dcb80c-7e8b-11e8-9732-e8ee0abd66fa.png)

**Comment by benz145** [comment]
Ok thanks I'll do that -- if I do though, will Windows + Tab still activate ZVD Desktop View, or will it fall back to the default Windows version of Desktop View?

**Comment by mzomparelli** [comment]
No, all hotkeys will stop working. The profile idea I mentioned would be perfect for you. For now you will have to delete the hotkeys you don't want. Sorry.

**Comment by mzomparelli** [comment]
I mean all zVD hotkeys will stop working. Default Windows hotkeys will function while zVD’s will not.
